docker/docker-compose.yml

```diff
@@ -93,3 +93,54 @@ services:
       - GPC_CONSUMER_SDS_URL
       - GPC_CONSUMER_SDS_APIKEY
       - GPC_ENABLE_SDS
+
+  gpconnect-db:
+    image: mariadb
+    networks: 
+      - gpconnect-network-1.5
+    environment:
+      - MYSQL_ROOT_PASSWORD=password
+      - MYSQL_USER=gpconnectdbuser
+      - MYSQL_PASSWORD=gpc0nn3ct
+      - MYSQL_DATABASE=gpconnect1_5
+    command: [
+        '--wait_timeout=28800',
+    ]
+
+  gpconnect-api:
+    image: nhsdigitalmait/gpconnect:1.5.1
+    restart : always
+    ports:
+        - 5000:19191
+    networks: 
+      - gpconnect-network-1.5
+    environment:
+      - TZ=Europe/London
+      - DATABASE_ADDRESS=gpconnect-db
+      - DATABASE_USERNAME=gpconnectdbuser
+      - DATABASE_PASSWORD=gpc0nn3ct
+      - DATABASE_SCHEMA=gpconnect1_5
+      - SERVER_BASE_URL=https://data.developer.nhs.uk/B82617/STU3/1/gpconnect/fhir
+      - CONTEXT_PATH=/B82617/STU3/1/gpconnect/
+    links:
+      - gpconnect-db
+    depends_on:
+      - gpconnect-db
+
+  tkw:
+    image: nhsdigitalmait/tkw_gpconnect_simulator:1.5.1
+    restart : always
+    ports:
+        - 4854:4854
+    networks: 
+      - gpconnect-network-1.5
+    environment:
+      - TZ=Europe/London
+    links:
+      - gpconnect-api
+    depends_on:
+      - gpconnect-api
+    
+networks: 
+  gpconnect-network-1.5:
+    driver: bridge
```

Comment:
Should we use the same network for GPC services as for GP2GP? Not sure if the containers are going to be able to talk to each other.